src/ts/components/account/GeneratePhrase.tsx

```diff
@@ -14,37 +14,67 @@ interface IGeneratePhraseProps extends StateProps, DispatchProps, RouteComponent
 
 interface IGeneratePhraseState {
   mnemonic: string,
-  errorMessage?: string
+  message?: string,
+  color: 'blue'|'red'
 }
 
 class GeneratePhrase extends React.Component<IGeneratePhraseProps, IGeneratePhraseState> {
 
   state: IGeneratePhraseState = {
-    mnemonic: ''
+    mnemonic: '',
+    color: 'blue'
   }
 
   constructor (props) {
     super(props)
 
-    this.selectAll = this.selectAll.bind(this)
-    this.handleClick = this.handleClick.bind(this)
-
     // generate the mnemonic
     generateMnemonic().then(phrase => {
       this.setState({ mnemonic: phrase })
     })
   }
 
-  handleClick () {
-    this.setState({ errorMessage: '' })
+  handleClick = () => {
+    this.setState({ message: '' })
     this.props.setNewPhrase(this.state.mnemonic)
     this.props.history.push(CONFIRM_PHRASE_ROUTE)
   }
 
-  selectAll (event) {
+  selectAll = (event) => {
     event.target.select()
   }
 
+  copyText = () => {
+    const el = document.createElement('textarea')
+    el.value = this.state.mnemonic
+    el.setAttribute('readonly', '')
+    el.style.position = 'absolute'
+    el.style.left = '-9999px'
+    document.body.appendChild(el)
+    el.select()
+    document.execCommand('copy')
+    document.body.removeChild(el)
+
+    this.setState({message: t('copyTextMessage')})
```

Comment:
I can't fine `copyTextMessage` in message. Are we plan to display something like `copying...` ?